prophesee_ros_driver/launch/prophesee_publisher.launch

```diff
@@ -3,17 +3,27 @@
   <node pkg="prophesee_ros_driver" type="prophesee_ros_publisher" name="prophesee_ros_publisher" respawn="false" output="screen">
     <param name="camera_name" value="camera" />
     <param name="publish_cd" value="true" />
-    <param name="publish_graylevels" value="false" />
+    <param name="publish_graylevels" value="true" />
     <param name="publish_imu" value="true" />
 
     <!-- Path to the file with the camera settings -->
-    <param name="bias_file" value="" />
+    <!--param name="bias_file" value="$(find prophesee_ros_driver)/cfg/gen3_low_sensitive_settings.bias" /-->
 
-    <!-- Maximum event rate in kEv/s -->
-    <param name="max_event_rate" value="6000" />
+    <!-- Higher sensitivity bias file -->
+    <param name="bias_file"  value="/usr/share/prophesee_driver/biases/gen3_EM.bias" />
+
+    <!-- Maximum event rate in kEv/s. With 0 does not set any limit-->
+    <param name="max_event_rate" value="0" />
```

Comment:
It works realtime with the activity filter?
In case @nlyubova I think we can maintain this value. What do you think?